std/system.d

```diff
@@ -14,7 +14,20 @@ immutable
 {
     /++
         Operating system.
-
+        
+        Example:
+        ```
```

Comment:
Unfortunately the Markdown syntax isn't supported by DDoc - have a look at this page:

https://dlang.org/spec/ddoc.html

or other files in Phobos. I looks roughly like this:

````
Example:
----
struct A
{
    int a;
    mixin(bitfields!(
        uint, "x",    2,
        int,  "y",    3,
        uint, "z",    2,
        bool, "flag", 1));
}
A obj;
obj.x = 2;
obj.z = obj.x;
----
```